src/m365/pp/commands/managementapp/managementapp-add.ts

```diff
@@ -0,0 +1,130 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import Utils from '../../../../Utils';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  appId?: string;
+  objectId?: string;
+  name?: string;
+}
+
+class PpManagementappAddCommand extends GraphCommand {
```

Comment:
Let's rename the class to `PpManagementAppAddCommand` to improve the readability.